### DISCUSSION TITLE
Mounting project drives to VMs

### DISCUSSION THREAD

**Discussion by manuGil** [TU-Delft-DCC/TU-Delft-DCC.github.io#4] [Archive]
How do I mount a project drive in a VMs to transfer data back and forward?

This is how: https://gitlab.tudelft.nl/acryan/data-management-for-researchers/-/wikis/Mount-Project-Drive-in-TU-Delft-VPS

**Comment by manuGil** [comment]
I couldn't connect following the instructions above. The docs is also skipping some details, e.g. the `fstab` file must be in the `/etc/` directory. 
When attempting to mount with `$sudo mount /media/<project-drive>`, the follow error is reported:

```bash
mount: /media/africarain: bad option; for several filesystems (e.g. nfs, cifs) 
you might need a /sbin/mount.<type> helper program.
```
@ebrahimifard any idea what I did wrong?

**Reply by mmaschenk** [reply]
@manuGil : this probably means that the packages for mounting cifs-type filesystems haven't been installed. Depending on your linux flavour you will need to install them first using

`% sudo apt install cifs-utils`

on ubuntu/debian like linuxes, or

`% sudo yum install cifs-utils`

on redhat/centos/fedora flavours.

**Reply by manuGil** [reply]
Thanks @mmaschenk. Things worked as expected after installing `cifs-util`

Though, I just realized this mounting approach is not what I am looking for, because mounting the storage requires authentication using the netID. Now I wonder if there's a way to mount the remote storage independently of netIDs. This is probably something I have to request via self service portal.

**Comment by mmaschenk** [comment]
@aecryan Suggested change to https://gitlab.tudelft.nl/acryan/data-management-for-researchers/-/wikis/Mount-Project-Drive-in-TU-Delft-VPS:

Step 3 (find your user id) can be accomplished by the linux command `id`. This outputs both your user and group id(s).

I tried to make a merge request for this change, but it turns out gitlab forks don't include the wiki in a project :-(

**Reply by aecryan** [reply]
@mmaschenk  nice suggestion! I will indeed update the guide with this info and re: installing cifs-util to make sure things work as expected.
